docs/topics/Voice_Connections.md

```diff
@@ -1,89 +1,189 @@
+```json
+{
+	"token": "my_token",
+	"guild_id": "41771983423143937",
+	"endpoint": "smart.loyal.discord.gg"
+}
+```
+
+With this information, we can move on to establishing a voice websocket connection.
 
-If our request succeeded, the gateway will respond with _two_ events (meaning your library must properly wait for both events before continuing), a [Voice State Update](#DOCS_GATEWAY/voice-state-update) _event_ (not a payload), and a [Voice Server Update](#DOCS_GATEWAY/voice-server-update) event. The first will contain a new key, `session_id` and the second will provide voice server information we can use to establish a new voice connection. With this information, we can move on to [Establishing a Voice Websocket Connection](#DOCS_VOICE_CONNECTIONS/establishing-a-voice-websocket-connection).
+## Establishing a Voice Websocket Connection
 
-### Establishing a Voice Websocket Connection
+Once we retrieve a session\_id, token, and endpoint information, we can connect and handshake with the voice server over another secure websocket. Unlike the gateway endpoint we receive in an HTTP [Get Gateway](#DOCS_GATEWAY/get-gateway) request, the endpoint received from our [Voice Server Update](#DOCS_GATEWAY/voice-server-update) payload does not contain a URL protocol, so some libraries may require manually prepending it with "wss://" before connecting. Once connected to the voice websocket endpoint, we can send an [OP 0 Identify](#DOCS_VOICE_CONNECTIONS/voice-events-voice-op-codes) payload with our server\_id, user\_id, session\_id, and token:
 
-Once we retrieve a session_id, token, and endpoint information, we can connect and handshake with the voice server over another secure websocket. Unlike the gateway endpoint we receive in a HTTP [get gateway](#DOCS_GATEWAY/get-gateway) request, the endpoint received from our [Voice Server Update](#DOCS_GATEWAY/voice-server-update) payload does not contain a URL protocol, so some libraries may require manually prepending it (`wss://`) before connecting. Once connected to the voice websocket endpoint, we can send a OP0 Identify with our server id, user id, session id, and token.
+>warn
+>Much like our standard Gateways, voice sockets support versioning via querystring parameters. To ensure that you have the most up-to-date information, please append "`?v=3`" to your URL. Otherwise, we cannot guarantee that the [OP codes](#DOCS_VOICE_CONNECTIONS/voice-events-voice-op-codes) documented here will reflect what you receive over the socket.
```

Comment:
we need to make a version table here, similar to the others.. im surprised we never documented this